src/test/resources/features/multiple_choice_question.feature

```diff
@@ -0,0 +1,115 @@
+    When I add a question
+    And I select "Multiple" question type
+    When I type question text "Text question is here" into "Q1"
+    And I wait for 2 sec
+    And I type "shbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgc" as option "Option 1*" into "Q1"
+    And I wait for 2 sec
+    And I type "shbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgcshbvlsvhydvgc" as option "Option 2*" into "Q1"
+    And I wait for 2 sec
+    And I select correct option in "Q1" as "Option 1*"
+    And I wait for 1 sec
+    When I click on "Save" button
+    And I wait for 1 sec
+#      Known issus max+1 characters ASKJ-
+    Then I verify that error message contains "Max 1000 characters allowed"
+
+  @multiple5
+  Scenario: Verify that max 15 choices allowed
+    When I click on "Create New Quiz" button
+    And I wait for 1 sec
+    And I type quiz title "Olena Quiz"
+    When I add a question
+    And I select "Multiple" question type
+    And I create quiz with 15 choices
+    When I click on "Save" button
+    And I wait for 1 sec
+    Then quiz "Olena Quiz" should be displayed on the list of quizzes
```

Comment:
All your Scenarios should end with the step I delete "quizName" from the list pf quizzes